.meshrc.yaml

```diff
@@ -39,13 +39,15 @@ sources:
             responseSchema: ./json-schemas/snapshotBackground.json
             responseTypeName: Background
           - type: Query
-            field: BulkDownloadDetails
-            path: /bulk_downloads/{args.bulkDownloadId}
```

Comment:
does `path` actually do anything, since we use custom resolvers? I've been curious about this...

Reply:
lol - yeah - if you don't have custom resolvers than thats where it sends the query and I think there is a difference in how the arguments come into the query (if they are mentioned in the path then they are just basic but if they are in the resolver they come in through the input argument) - I haven't tried just leaving it out for custom resolvers thou.